Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -782,9 +788,8 @@ Common::Vec3 Wiimote::GetTotalAngularVelocity() const
 
 Common::Matrix44 Wiimote::GetTotalTransformation() const
 {
-  return GetTransformation(Common::Matrix33::FromQuaternion(
-      m_imu_cursor_state.rotation *
-      Common::Quaternion::RotateX(m_imu_cursor_state.recentered_pitch)));
+  return GetTransformation(GetRotationalMatrix(m_imu_cursor_state.orientation),
```

Comment:
If the cursor rotation isn't used here where is it used?  Seems like you need to calculate orientation (for when sensor bar isn't enabled) up above.